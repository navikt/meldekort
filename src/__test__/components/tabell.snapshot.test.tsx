import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ProviderWrapper from '../testSetup/providerWrapper';
import Tabell from '../../app/components/tabell/desktop/tabell';

it('Tabell', () => {
  const rader = [{k: 1}, {k: 2}, {k: 3}]
  const kolonner = [
    {key: "En", label: <>En</>},
    {key: "To", label: <>To</>},
    {key: "Tre", label: <>Tre</>}
  ]

  const tree = renderer.create(
    <ProviderWrapper>
      <Tabell rows={rader} columns={kolonner} />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
