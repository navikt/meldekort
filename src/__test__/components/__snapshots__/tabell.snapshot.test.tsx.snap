// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabell 1`] = `
<div
  className="tabell"
>
  <div>
    <div
      style={
        Object {
          "overflow": "hidden",
          "position": "relative",
        }
      }
    >
      <table
        className="jsonTable"
      >
        <thead>
          <tr>
            <th
              className="jsonColumn jsonColumn_En"
              colSpan={1}
              data-key="En"
              onClick={[Function]}
              rowSpan={null}
            >
              En
            </th>
            <th
              className="jsonColumn jsonColumn_To"
              colSpan={1}
              data-key="To"
              onClick={[Function]}
              rowSpan={null}
            >
              To
            </th>
            <th
              className="jsonColumn jsonColumn_Tre"
              colSpan={1}
              data-key="Tre"
              onClick={[Function]}
              rowSpan={null}
            >
              Tre
            </th>
          </tr>
        </thead>
        <tbody>
          <tr
            className="jsonRow jsonEven"
            onClick={[Function]}
          >
            <td
              className="jsonCell jsonCell_En"
              data-key="En"
              onClick={[Function]}
            />
            <td
              className="jsonCell jsonCell_To"
              data-key="To"
              onClick={[Function]}
            />
            <td
              className="jsonCell jsonCell_Tre"
              data-key="Tre"
              onClick={[Function]}
            />
          </tr>
          <tr
            className="jsonRow jsonOdd"
            onClick={[Function]}
          >
            <td
              className="jsonCell jsonCell_En"
              data-key="En"
              onClick={[Function]}
            />
            <td
              className="jsonCell jsonCell_To"
              data-key="To"
              onClick={[Function]}
            />
            <td
              className="jsonCell jsonCell_Tre"
              data-key="Tre"
              onClick={[Function]}
            />
          </tr>
          <tr
            className="jsonRow jsonEven"
            onClick={[Function]}
          >
            <td
              className="jsonCell jsonCell_En"
              data-key="En"
              onClick={[Function]}
            />
            <td
              className="jsonCell jsonCell_To"
              data-key="To"
              onClick={[Function]}
            />
            <td
              className="jsonCell jsonCell_Tre"
              data-key="Tre"
              onClick={[Function]}
            />
          </tr>
        </tbody>
        <tfoot />
      </table>
    </div>
  </div>
</div>
`;
